src/legos/typography/Heading.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import styled from "styled-components";
+import { Heading as LibHeading } from "grommet";
+
+const getFontSize =(level)=> {
+  switch (level) {
+    case 1:
+      return '54px'
+      break;
+    case 2:
+      return '40px'
+      break;
+    case 5:
+      return '16px'
+      break;
+    default:
+      return '16px'
+      break;
+  }
+};
+
+const getFontWeight =(level)=> {
+  switch (level) {
+    case 1:
+      return '800'
+      break;
+    case 2:
+      return '300'
+      break;
+    default:
+      return '400'
+      break;
+  }
+};
+
+const getlineHeight =(level)=> {
```

Comment:
Consider installing [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) to your VS Code. My VS Code says that `getline` is unknown word.  
![image](https://user-images.githubusercontent.com/12502321/82296580-109e5480-99ba-11ea-885d-c5a448a416c1.png)
In this case you have to write that constant name in a camelCase: `getLineHeight`